src/kernel/src/arch/aarch64/interrupt.rs

```diff
@@ -167,7 +154,7 @@ pub fn allocate_interrupt_vector(
 // code for IPI signal to send
 // needed by generic IPI code
 pub enum InterProcessorInterrupt {
-    Reschedule = 0, /* TODO */
+    Reschedule = 2, /* TODO */
```

Comment:
What does this 2 mean and why it a magic number?

Reply:
2 is the reserved IPI number. This type is exposed to generic code, so we defined it in case it is used. As far as I can tell, however, it seems unused. x86 uses this to wake up cores in the absence of mwait.

Reply:
OK, let's leave it.